include/nuttx/net/can.h

```diff
@@ -43,6 +44,7 @@
 
 #include <nuttx/config.h>
 #include <nuttx/can.h>
+#include <nuttx/net/netconfig.h>
```

Comment:
why add this

Reply:
there is where net_stats_t type (needed in can_stats_s new structure) is defined